### DISCUSSION TITLE
How to use 1000 separator in format number

### DISCUSSION THREAD

**Discussion by longlee218** [exceljs/exceljs#2471] [Q&A]
## 1000 separator

I have a number value  like 1000 or greater than 1 thousand.  How to setting 1000 separtor format in number cell.

**Comment by Jadamae77** [comment]
I have the same issue. I would like to use "Comma Style" in my column, but can't figure out how to configure it.

**Comment by Jadamae77** [comment]
This can be done as follows: 

`style: { numFmt: '#,##0.00'}`

`#` = optional digits
